addons/l10n_il/data/account.group.template.csv

```diff
@@ -0,0 +1,11 @@
+id,code_prefix_start,code_prefix_end,name,chart_template_id/id
+il_group_100100,100100,100499,"רכוש קבוע",l10n_il.il_chart_template
```

Comment:
Should be in English and then translated to Hebrew

Reply:
@jco-odoo really? I thought it was ok to add untranslated items in localisation modules

Reply:
@mart-e But this one is multilang and the CoA is in English